src/Constraint/ForeignKeyConstraint.php

```diff
@@ -10,7 +10,9 @@
 class ForeignKeyConstraint extends Constraint
 {
     private ?string $foreignSchemaName = null;
+    /** @var object|string|null $foreignTableName */
```

Comment:
Are you sure it should anything but the nullable string? What object could it be?

Reply:
yes null its used in tests Mysql.